src/template.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Micro-Tasks</title>
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="assets/icons/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="assets/icons/favicon/favicon-16x16.png"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Nunito:400,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/cocogoose" />
+  </head>
+  <body>
+    <main class="flex justify-center w-screen ms:flex-col">
+      <section
+      class="todo-section flex flex-col md:self-center bg-white md:min-w-[600px] sm:mx-2 md:w-6/12"
+      >
+      <div class="flex justify-center bg-regal-beige">
+        <img class="w-[400px]" src="assets/microtasks-logo.png" alt="microtasks-logo" />
+      </div>
+        <div class="title flex justify-between w-full px-2 py-3">
+          <h3>Today's Todos</h3>
```

Comment:
Consider styling the title of the task to be different from the task items.

Making the title bold and bigger will make it stand out more as a title.

Reply:
I see that @PraisesPJMT mayybe I can disable or enable an input tag, but there is room for improvement, I agree 😄